src/tasks/install-application/install-application-v2/task.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "cd66049c-f04b-4be1-aac1-df4f5a7e51ad",
+  "name": "PowerPlatformInstallApplication",
+  "friendlyName": "Power Platform Install Applications",
+  "description": "(Preview): Power Platform Install Applications from AppSource",
+  "author": "Microsoft",
+  "helpMarkDown": "[More Info](https://aka.ms/buildtoolsdoc)",
+  "category": "Deploy",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": "0",
+    "Minor": "1",
+    "Patch": "0"
+  },
+  "preview": false,
```

Comment:
As `"(Preview): ..."` is in the description above, should this line be `"preview": true,` ?

Reply:
good catch, the preview in the description is stale text from this spring, when the API was preview. But it is GA now. Fixing description